data/ec/signature_interface.py

```diff
@@ -0,0 +1,8 @@
+def generate_signature_key_pair(security_param, randomness) ->
+    (secret_key, public_key)
+
+
+def sign(message, secret_key) -> signature
+
+
+def verify(message, signature, public_key) -> bool
```

Comment:
animation: add recover
2nd animation add ec in front of sign, verify and recover